x-pack/test/functional/apps/rollup_job/rollup_jobs.js

```diff
@@ -4,64 +4,70 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+
+import datemath from '@elastic/datemath';
 import expect from '@kbn/expect';
-import { indexBy } from 'lodash';
+import { mockIndices } from './hybrid_index_helper';
 
 export default function ({ getService, getPageObjects }) {
-  const esArchiver = getService('esArchiver');
-  const log = getService('log');
-  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'header']);
+
+  const es = getService('es');
+  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'indexManagement', 'settings', 'discover']);
 
   describe('rollup job', function () {
-    this.tags('smoke');
-    before(async () => {
-      // init data
-      await esArchiver.loadIfNeeded('logstash_functional');
-      await esArchiver.load('canvas/default');
+    const rollupJobName = 'rollup-to-be-' + Math.floor(Math.random() * 10000);
+    const indexName = 'rollup-to-be';
+    //make sure all dates have the same concept of "now"
+    const now = new Date();
+    const pastDates = [
+      datemath.parse('now-1d', { forceNow: now }),
+      datemath.parse('now-2d', { forceNow: now }),
+      datemath.parse('now-3d', { forceNow: now }),
+    ];
+
+    it('create new rollup job', async () => {
+      const indexPattern = 'to-be*';
+      const interval = '1000ms';
+
+      pastDates.map(async (day) => {
+        await es.index(mockIndices(day));
+      });
+
       await PageObjects.common.navigateToApp('rollupJob');
-    });
+      await PageObjects.rollup.createNewRollUpJob(rollupJobName, indexPattern, indexName,
+        interval, ' ', true, { time: '*/10 * * * * ?', cron: true });
 
-    after(async () => await esArchiver.unload('logstash_functional'));
+      await PageObjects.common.navigateToApp('indexManagement');
 
-    it('create and save a new job', async () => {
-      const jobName = 'Testjob1';
-      const indexPattern = '.kibana*';
-      const indexName = 'rollup_index';
-      const interval = '1000ms';
+      await PageObjects.indexManagement.toggleRollupIndices();
```

Comment:
I guess this should be another issue? @cjcenizal

Reply:
You could do it in this PR if you want, or in a separate PR. If the latter then I would make an issue to track it because personally I'd probably forget about it otherwise. :)

Reply:
https://github.com/elastic/kibana/issues/48280